### PR TITLE
General icon and cds cleanup

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

- Continuing the process of transitioning to CDS components
- Updates the sandbox with some component changes to Alert
- Updates the Select cds component to use cds button instead of IconButton
- Removal of some deprecated styled components as files are touched

References: [CV2-3360](https://meedan.atlassian.net/browse/CV2-3360), [CV2-3353](https://meedan.atlassian.net/browse/CV2-3353)

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Manual testing

## Things to pay attention to during code review
n/a

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [x] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

[CV2-3360]: https://meedan.atlassian.net/browse/CV2-3360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CV2-3353]: https://meedan.atlassian.net/browse/CV2-3353?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by brianfleming** [comment]
more of the same, just continuing to clean up stuff

File: src/app/components/team/SwitchTeamsComponent.js [hunk 1]
```diff
@@ -157,9 +159,14 @@ class SwitchTeamsComponent extends Component {
...
                 <ListItemSecondaryAction>
-                  <IconButton edge="end" onClick={this.setCurrentTeam.bind(this, team, currentUser)}>
-                    <KeyboardArrowRight />
-                  </IconButton>
+                  <ButtonMain
+                    iconCenter={<KeyboardArrowRight />}
+                    theme="text"
+                    variant="text"
+                    size="default"
+                    edge="end"
```
Comment (amoedoamorim) [inline src/app/components/team/SwitchTeamsComponent.js:167]: is edge a ButtonMain prop?

**Review by amoedoamorim — APPROVED** [review]

File: src/app/components/team/SwitchTeamsComponent.js (hunk 1 above)
Comment (brianfleming) [reply src/app/components/team/SwitchTeamsComponent.js:167]: good catch @amoedoamorim 
Thanks for the review
I missed this when converting. I will remove once (hopefully) the integrations test pass

Comment (brianfleming) [reply src/app/components/team/SwitchTeamsComponent.js]: ok removed, integration tests didnt pass for other reasons, but this was resolved

**Pull request merged**